book/graphs/pharmacokinetic/pkpg04.qmd

```diff
@@ -23,7 +25,7 @@ adpp <- synthetic_cdisc_dataset("latest", "adpp")
 
 ## Plot of CMAX by Visit
 
-```{r, warning=FALSE}
+```{r plot1, warning=FALSE, test = list(plot_v1 = "plot")}
```

Comment:
Need to add seed to some plots where some randomness is generated like plot text label using `geom_text_repel()`
<img width="477" alt="Screenshot 2023-07-21 at 12 17 08 PM" src="https://github.com/insightsengineering/tlg-catalog/assets/49812166/eddd0de3-77e3-4c12-b9ed-4d331ad1f15a">